css-layout-classes/styles.css

```diff
@@ -0,0 +1,28 @@
+body {
```

Comment:
If you add a rule set like this, your padding will not add to the width of your elements, and the text should fall in the right place.

```css
* {
  box-sizing: border-box;
}
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing